File: pgm/ABC_Visual_Basic/ABC_150_A.vb
Public Class Main
    Public Shared Sub Main(ByVal args() As String)
        Dim K, X as Integer
        Dim arr(2) as String
        arr = Split(Console.ReadLine())
        K = arr(0) : X = arr(1)
        Console.WriteLine(if(X <= K * 500, "Yes", "No"))
    End Sub
End Class

File: pgm/ABC_C#/ABC_150_A.cs
using System;

class Program
{
    static void Main()
    {
        String[] arr = Console.ReadLine().Split();
        int K = int.Parse(arr[0]), X = int.Parse(arr[1]);
        Console.WriteLine(X <= 500 * K ? "Yes" : "No");
    }
}